### PR TITLE
Hello World samples for Java 11

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+`entrypoint` element in the [app.yaml](app.yaml) to start your application. The
+sample uses the `java` command is used to compile and execute the Java source code.
+
+## Setup
+See [Prerequisites](../README.md#Prerequisites).
+
+## Deploy to App Engine Standard
+
+```
+gcloud app deploy
```
Comment (kurtisvg) [inline appengine-java11/hello-world/README.md]: In one of the other samples it suggested using the mvn command line -- should we be pushing gcloud instead? Seems like we should be consistent.

File: appengine-java11/http-server/pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example.appengine</groupId>
+  <artifactId>http-server</artifactId>
+  <version>0.1</version>
+  <properties>
```
Comment (kurtisvg) [inline appengine-java11/http-server/pom.xml]: Please add the parent POM (And run formatting on this file)

File: appengine-java11/hello-world/README.md (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/hello-world/README.md]: This is the most basic sample and doesn't use maven. Documentation will be used to explain the flexibility of the runtime.

File: appengine-java11/http-server/pom.xml (hunk 2 above)
Comment (averikitsch) [reply appengine-java11/http-server/pom.xml]: done.

**Comment by kurtisvg** [comment]
@averikitsch tests on this one didn't run either

**Comment by averikitsch** [comment]
@kurtisvg can these samples be merged now?

**Pull request merged**